File: GameLibrary/src/cc/lib/probot/Level.java
package cc.lib.probot;

import cc.lib.utils.Reflector;

public class Level extends Reflector<Level> {

    static {
        addAllFields(Level.class);
    }

    public String label = "<UNNAMED>";
    public String info = "<EMPTY>";
    public Type [][] coins = { { Type.EM } };
    public Boolean [] lazers = { true, true, true };
    public int numJumps = 0;
    public int numTurns = -1;
    public int numLoops = -1;
}

File: Probot/src/cc/lib/probot/Level.kt
package cc.lib.probot

import cc.lib.reflector.Reflector

class Level : Reflector<Level>() {
	companion object {
		init {
			addAllFields(Level::class.java)
		}
	}

	@JvmField
    var label = "<UNNAMED>"
	@JvmField
    var info = "<EMPTY>"
	@JvmField
    var coins = arrayOf(arrayOf(Type.EM))
	@JvmField
    var lazers = arrayOf(true, true, true)
	@JvmField
    var numJumps = 0
	@JvmField
    var numTurns = -1
	@JvmField
    var numLoops = -1
}